src/Microsoft.AspNetCore.Mvc.Razor.Host/Internal/ViewComponentTagHelperChunkVisitor.cs

```diff
@@ -58,39 +48,33 @@ public override void Accept(Chunk chunk)
             var parentChunk = chunk as ParentChunk;
             var tagHelperChunk = chunk as TagHelperChunk;
 
-            if (parentChunk != null && !(parentChunk is TagHelperChunk))
+            if (parentChunk != null)
```

Comment:
This doesn't work does it? I'd think you'd have to check if tagHelperChunk != null first